server/blob/blob-cassandra/pom.xml

```diff
@@ -65,6 +65,10 @@
             <groupId>${james.groupId}</groupId>
             <artifactId>james-server-util</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>metrics-logger</artifactId>
```

Comment:
I don't like that... at all... but I had to add something for CassandraBlobStoreFactory that is not in the test package and is being used by other test classes in other modules... Not sure what would be the best way here... or add the metrics as a param in forTesting method?

Reply:
How about moving the forTesting stuff in tests, and fix imports of tests that depends on it?

> or add the metrics as a param in forTesting method?

Good solution and likely easier so :+1: